crates/loader/src/local/mod.rs

```diff
@@ -167,7 +167,13 @@ async fn prepare(
     let variables = raw
         .variables
         .into_iter()
-        .map(|(key, var)| Ok((key, var.try_into()?)))
+        .map(|(key, var)| {
+            Ok((
+                key.clone(),
+                var.try_into()
+                    .map_err(|err| anyhow!("variable '{}': {}", key.clone(), err))?,
```

Comment:
This last call to `key.clone()` shouldn't be be necessary

Reply:
👍